ansible/configs/ocp4-cluster/default_vars.yml

```diff
@@ -37,6 +37,9 @@ student_name: lab-user
 # Mutually exclusive with {{ install_student_user }}
 install_ipa_client: false
 
+# The domain that you want to add DNS entries to
+cluster_dns_zone: ocp.example.com
```

Comment:
This should come from the secret file. Would be good to comment this out - and add a comment that this needs to come from the deployment secret.

Reply:
ok